next.config.js

```diff
@@ -83,6 +83,56 @@ module.exports = withNextra({
         destination: "/recipes/prisma",
         permanent: false,
       },
+      {
+        source: "/references/preview-databases",
+        destination: "/core-concepts/deploy#preview-databases-on-snaplet-cloud",
+        permanent: false,
```

Comment:
This is a 302 at the moment, we need to change to 301, with permanent: true please